src/test/starknet.t.sol

```diff
@@ -124,6 +127,9 @@ contract StarknetTests is GoerliDssSpellTestBase, ConfigStarknet {
 
         assertEq(daiBridge.wards(addr.addr("MCD_PAUSE_PROXY")), 1, "StarknetTest/pause-proxy-not-ward-on-dai-bridge");
         assertEq(daiBridge.wards(addr.addr("MCD_ESM")),         1, "StarknetTest/esm-not-ward-on-dai-bridge");
+
+        uint256 l2DaiBridge = daiBridge.l2DaiBridge();
```

Comment:
This variable is unused and can be removed.

Reply:
Agree. Should I provide a PR or you will do the editing? @The-Arbiter